openshift/patch.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Expected usage is as an Helm post renderer.
+# Example usage:
+#   $ helm install my-release camunda/camunda-platform --post-renderer ./patch.sh
+#
+# This script is a Helm chart post-renderer for users on Helm 3.2.0 and greater. It allows removing default
+# values set in sub-charts/dependencies, something which should be possible but is currently not working.
+# See this issue for more: https://github.com/helm/helm/issues/9136
+#
+# The result of patching the rendered Helm templates is printed out to STDOUT. Any other logging from the
+# script is thus sent to STDERR.
+#
+# Note to contributors: this post-renderer is used in the integration tests, so make sure that it can be used
+# from any working directory.
+
+set -o pipefail
+
+# Perform two passes: once for single quotes, once for double quotes, as it's not specified that string values are
+# always output with single or double quotes
+sed -e "s/'@@null@@'/null/g" | sed -e 's/"@@null@@"/null/g'
```

Comment:
sed supports basic regex, so we can do:
```
sed  "s/['\"]@@null@@['\"]/null/g"
```

and if we like it simple, we can provide the 2 patterns to single sed as:
```
sed -e "s/'@@null@@'/null/g" -e 's/"@@null@@"/null/g'
```

Reply:
Sounds good. Will this behave the same on BSD platforms as well? I've had surprises with basic utilities on these :smile: I have no strong preference, I think all of these are equally fine, and this is definitely not a performance bottleneck. Do you have any preference?

Reply:
Then second one (the 2 patterns) to avoid any compatibility issues across the system :100: